dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py

```diff
@@ -129,10 +130,19 @@ def write_data_to_bigquery(
 
 def download_fb_page_insight_data_upload_to_bigquery():
     fb = FBPageInsight()
-    page_insight: PageWebInsightData = fb.get_page_default_web_insight()
-
     client = init_bigquery_client()
 
+    page_insight: PageWebInsightData = fb.get_page_default_web_insight()
```

Comment:
forgive my ignorance, why we use `get_page_default_web_insight` here but using `get_page_default_web_insight(period=Period.day)` at Line 143?

and they are all written into the same table ? 😂

Reply:
The first one is to query period=Period.week (default) and this one is for Period.day. Thare are period and end_time in the same table. For example, end_time=(2021,8,30) and period='week" means this data is  accumulated from 
```
start =(2021,8.30) - week = (2021, 8,23)
end  = (2021,8,30)
```
In FB query design, you can specify end_time and period.

Reply:
Thanks for the explanation!